### ISSUE TITLE
NameError: name 'os_platform' is not defined @my windows10

### ISSUE THREAD

**Issue by chungyan5** [cjnaz/rclonesync-V2#27]
hi, 

I just download the latest master branch code, and run at my windows 10, and got this error. 
```
2019-07-07 09:37:29,277:  ***** BiDirectional Sync for Cloud Services using rclone *****
Traceback (most recent call last):
  File "C:\Portable\rclonesync\rclonesync.py", line 711, in <module>
    path2_base = pathparse(args.Path2)
  File "C:\Portable\rclonesync\rclonesync.py", line 678, in pathparse
    if path[1] == ':' and os_platform == 'Windows':     # Windows drive letter case
NameError: name 'os_platform' is not defined
```

Then, i added this line of code at line 710
```
...    
710:    os_platform = platform.system()                             # Expecting 'Windows' or 'Linux'
711:    path1_base = pathparse(args.Path1)
...
```
This issue seems settled, i do not exactly know is it right code. Please review. 

thanks
chungyan5

**Comment by silenceleaf** [comment]
same error here, move this line up will solve this issue. Please merge this to master.

**Comment by cjnaz** [comment]
Agreed.   I introduced the error in a recent release and I do my thorough
testing on Linux.

**Comment by cjnaz** [comment]
Fixed in V2.8.  Thanks.
